Source/WebCore/animation/WebAnimation.cpp

```diff
@@ -1907,12 +1907,22 @@ void WebAnimation::setBindingsRangeEnd(TimelineRangeValue&& rangeEndValue)
         effect->animationRangeDidChange();
 }
 
-void WebAnimation::setRange(TimelineRange range)
+void WebAnimation::setRangeStart(SingleTimelineRange range)
 {
-    if (m_timelineRange == range)
+    if (m_timelineRange.start == range)
         return;
 
-    m_timelineRange = range;
+    m_timelineRange.start = range;
+    if (RefPtr effect = this->effect())
+        effect->animationRangeDidChange();
+}
+
+void WebAnimation::setRangeEnd(SingleTimelineRange range)
```

Comment:
The name `rangeEnd` would be better for the parameter.

Reply:
Fixed.